### PR TITLE
REF/ENH: designer widgets via entrypoints

### PR DESCRIPTION
**Pull request by klauer**
## Description
* Use the new entrypoint support in PyDM to export our widgets to the qt designer
* Marked as WIP as the categories are not yet set for the widgets and the original `plugin.py` remains

## Motivation and Context
* My local environment isn't as good as our deployed pcds-env and I don't want to have to tweak PYQTDESIGNERPATH any more

## How Has This Been Tested?
`pip install -e` and `Designer`

### CODE REVIEW

**Comment by klauer** [comment]
(Restarting CI)

**Comment by klauer** [comment]
I think this should be good now:
image

**Review by tangkong — APPROVED** [review]
LGTM.  As Ken knows I didn't get this working locally, but that's a me problem and shouldn't hold up the merge

**Comment by ZLLentz** [comment]
The implementation looks correct
This is always a pain to double-check locally, so instead I'll trust Ken's test and compare against the widgets available before this change as a quick completeness sanity check:

image

Compared to previous, the new implementation:
- fixes the non-alphabetical order
- moves the gauges into the symbols category

I think I'll take a moment to find the bit that miscategorizes the gauges but the rest looks good

File: pcdswidgets/vacuum/gauges.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
-from qtpy.QtCore import QSize
```
Comment (ZLLentz) [inline pcdswidgets/vacuum/gauges.py:1]: This file missing _qt_designer_ attributes to preserve the "PCDS Gauges" category

Comment (klauer) [reply pcdswidgets/vacuum/gauges.py:1]: Good catch 👍

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**